firebase-emulator/fixtures/sample-imageswipe-test/saved-data/auth_export/accounts.json

```diff
@@ -0,0 +1 @@
+{"kind":"identitytoolkit#DownloadAccountResponse","users":[{"localId":"2uGhB2WNfipw4eiiLiovjxzjY24f","createdAt":"1635148124461","lastLoginAt":"1635148124461","emailVerified":false,"email":"test@test.com","salt":"fakeSaltxvJVLUvcTpxVb1qzMmoP","passwordHash":"fakeHash:salt=fakeSaltxvJVLUvcTpxVb1qzMmoP:password=password","passwordUpdatedAt":1635148124461,"validSince":"1635148124","providerUserInfo":[{"providerId":"password","email":"test@test.com","federatedId":"test@test.com","rawId":"test@test.com","displayName":"test"}],"lastRefreshAt":"2021-10-25T07:48:44.462Z","displayName":"test"}]}
```

Comment:
might be good to pretty print this one

Reply:
Hey! Not sure what you mean by that. Would it be possible to give an example?

Reply:
hey, sorry for late reply, just meant not all on one line that's all, not critical, just a suggestion 🙂